lib/serverless.js

```diff
@@ -215,6 +220,49 @@ class Serverless {
   logDeprecation(code, message) {
     return this._logDeprecation(`EXT_${ensureString(code)}`, ensureString(message));
   }
+
+  extendConfiguration(configurationPathKeys, value) {
+    ensureArray(configurationPathKeys, {
```

Comment:
Let's reassign `configurationPathKeys` with `ensureArray` result (it normalizes values to strings when applicable)